civiwiki/frontend/static/views/friends_list_view.js

```diff
@@ -0,0 +1,103 @@
+var FriendsListView = Backbone.View.extend({
+    el: '#friends', 
+
+    friendsTemplate: _.template($('#friends-template').html()),
+
+    initialize: function(options){
+      var _this = this; 
+
+      options = options || {}; 
+
+      _this.user_id = options.user_id;
+      _this.friend_requests = options.friend_requests;
```

Comment:
NOPE I LIED THIS IS A DIFFERENT VIEW.  You killed it.  The only thing you need to change is this: Instead of `_this.friend_requests = options.friend_requests` do `_this.friend_requests = new Backbone.Collection(_this.userModel.toJSON().friend_requests)` because you can't listen for change events unless it is a `collection` or a `model` and `toJSON().friend_requests` removes the backbone from it